_events/discussions/event-013.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Research Software Directories: What, Why and How?"
+authors:
+    - &speaker
+      name: Mark Woodbridge
+      orcid: 0000-0002-0650-9751
+      is_speaker: true
+      affiliation: 1
+    - &speaker
+      name: Vanessa Sochat
```

Comment:
```suggestion
    - name: Vanessa Sochat
```
from the YAML-syntax there cannot be duplicated assignments of the ``&speaker`` variable. It should only be the first one for technical reasons.